packages/components/src/Checkbox/Checkbox.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import { generateSnapshots } from "@chanzuckerberg/story-utils";
+import { render } from "@testing-library/react";
+import React from "react";
+import Checkbox from "./Checkbox";
+import * as CheckboxStoryFile from "./Checkbox.stories";
+
+describe("<Checkbox />", () => {
+  generateSnapshots(CheckboxStoryFile);
+
+  test("throws an error if no label or aria-label", () => {
+    expect(() => {
+      render(<Checkbox />);
+    }).toThrow(/must provide a visible label or aria-label/);
```

Comment:
Is this still annoyingly printing this error message when the test runs? Could spyOn `console.error` if it is.

Reply:
nope! throwing the error fixed it 👍